opentelemetry-distro/setup.cfg

```diff
@@ -41,7 +41,7 @@ packages=find_namespace:
 zip_safe = False
 include_package_data = True
 install_requires =
-    opentelemetry-api ~= 1.3
+    opentelemetry-api == 1.4.0.dev0
```

Comment:
What's the reason for this change?

Reply:
I didn't realize we were pinning version specifically as of #1933 ... I thought this was just missed in a release 😅 . Will revert this change!